.travis.yml

```diff
@@ -13,7 +12,7 @@ env:
 cache:
   apt: true
 
-matrix:
+jobs:
```

Comment:
What does this do? (matrix -> jobs)

Reply:
`matrix` has been deprecated to an alias as of dpl v2.

The new key is called jobs: https://config.travis-ci.com/

Did the same thing in Aikido when I did the Travis updates: https://github.com/personalrobotics/aikido/commit/e3ab367db8889575c07b25355ec35e91b1585aaa